github-for-jira.sd.yml

```diff
@@ -113,8 +107,28 @@ environmentOverrides:
         INSTANCE_NAME: github-for-jira-stg
         # * Commented out env vars are to be saved in secrets
         # WEBHOOK_SECRET: for signed signature when syncing an org
+    resources:
+      - type: globaledge
+        name: proxy
+        attributes:
+          default_vanity_dns: false
+          domain:
+            - github.stg.atlassian.com
+          upstream_address:
```

Comment:
So I guess we just remove the `upstream_address` when we want to make the switch and then the traffic goes to our service?

Reply:
I think only one of the globaledge can create the `default_vanity_dns`.  There's a couple of options from here on how to switch over which we can discuss once the migration is done :)